include/RAJA/exec-agency/raja_agency.hxx

```diff
@@ -78,7 +78,10 @@ namespace RAJA
 //
 
 template <typename AGENT, typename WORKER>
-struct agency_base { };
+struct agency_base { 
+    using Agent_t = AGENT;
```

Comment:
This lets us pull these out more easily when we can't just add them to the template (see the tests)